nginx/plan.ps1

```diff
@@ -1,12 +1,12 @@
 $pkg_name="nginx"
 $pkg_origin="core"
-$pkg_version="1.15.3"
+$pkg_version="1.15.4"
```

Comment:
Uhm.. wouldn't we want 1.15.5 here, too?

Reply:
@srenatus This PR is getting so old, I'm having to update it as new nginx versions are release, and in this case I missed the windows one. Fixed now.

Reply:
👍 thank you!